modules/developer-cli-odo-creating-a-java-microservice-jpa-application.adoc

```diff
@@ -0,0 +1,104 @@
+= Creating a Java MicroServices JPA application
+
+With `odo`, you can create and manage a sample Java MicroServices JPA application.
+
+.Procedure
+
+. Clone the sample application:
++
+[source,terminal]
+----
+$ git clone https://github.com/OpenLiberty/application-stack-samples.git
+----
```

Comment:
In my opinion, being a developer knows how to do a `git clone`. So is it necessary to show this command?

Feel free to neglect this comment if it doesn't go with your scope or pitch of this file.

Reply:
@rishumehra since we need to provide a specific URL for the app, I felt like it would be straightforward to display it with the command for copy-n-pasting purposes.

Reply:
@boczkowska Then I think it is fine. I gave my opinion because the `git clone` command or concept is very basic for the developers. I didn't notice the specific URL of the app thing earlier, thanks for bringing that up. :)